src/molderl_utils.erl

```diff
@@ -147,5 +156,16 @@ gen_messagepacket_empty_test() ->
     Expected = <<<<"foo">>/binary, 23:64, 0:16>>,
     ?assertEqual(Expected, Observed).
 
+%% -----------------------
```

Comment:
I think it would be more elegant here to have 1 function with an extra argument rather than two functions, eg
gen_process_name(stream, abcd)
and
gen_process_name(recovery, abcd)
instead of
gen_streamprocessname(abcd)
and
gen_recoveryprocessname(abcd)